### PR TITLE
Personal/issue06

### LINKED ISSUE #15: [T06] Create Game GUI
Create GUI wrapper for game_logic and connections, with full functionality to allow for multiplayer gaming.

### PR DESCRIPTION
**Pull request by krzysztofzajaczkowski**
Create game GUI and handle game events, aswell as connection errors and end game.

### CODE REVIEW

File: client/App.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+#         app.label_var.set(new_label_name)
+# except KeyboardInterrupt as e:
+#     pass
+
+# # for i in range(5000):
+# #     print(i)
+
+# app.callback()
+
+# # app.root.destroy()
```
Comment (Debskij) [inline client/App.py]: Remove commented code

File: client/ConnectionThread.py [hunk 2]
```diff
@@ -21,61 +36,113 @@ def run(self):
...
+                        msg = self.handler_status.msg_to_send
+                    self.lock.release()
+                    sent = self.conn.send(str(msg).encode())
+                    break
+                except socket.error:
+                    if self.lock.locked():
+                        self.lock.release()
+                    if i == 5:
+                        self.logger.debug("Couldn't send message. Receiver must've closed connection!")
+                        self.handler_status.is_connected = "connection_error"
```
Comment (Debskij) [inline client/ConnectionThread.py:88]: i'd suggest self.lock.release() in finally block. More pythonic way

File: client/game_logic.py [hunk 3]
```diff
@@ -160,7 +161,7 @@ def set_package(self, package: str):
                 self.package_bin = package
                 self.package_change = True
         except KeyboardInterrupt:
-            print("ERROR!")
+            # print("ERROR!")
```
Comment (Debskij) [inline client/game_logic.py]: Remove commented code

File: client/game_logic.py [hunk 4]
```diff
@@ -235,3 +235,10 @@ def load_state(self):
         if self.package_parsed[4] == '01':
             self.end = True
 
+
+if __name__ == '__main__':
+    # jakis_lock = threading.Lock()
+    # game = Game([], jakis_lock)
+    # game.start()
+    gv = GameView()
+    gv.start()
```
Comment (Debskij) [inline client/game_logic.py]: Not used in normal application. Exists only for testing purpose. I'd suggest to remove it

File: client/App.py (hunk 1 above)
Comment (krzysztofzajaczkowski) [reply client/App.py]: Done

File: client/ConnectionThread.py (hunk 2 above)
Comment (krzysztofzajaczkowski) [reply client/ConnectionThread.py:88]: To be honest, I'm not sure why, but this suggestion results in some more response time needed. That's why I've done it this way. After discussion, we decided to leave it as it is.

File: client/game_logic.py (hunk 4 above)
Comment (krzysztofzajaczkowski) [reply client/game_logic.py]: Done

File: client/game_logic.py (hunk 3 above)
Comment (krzysztofzajaczkowski) [reply client/game_logic.py]: Done

**Pull request merged**